src/components/App/index.tsx

```diff
@@ -1,19 +1,11 @@
-import CardGrid from './CardGrid';
-import QRGenerator from './QRGenerator';
-import PuzzleGenerate from './PuzzleGenerate';
-import PuzzleInput from './PuzzleInput';
-import MapRenderer from './MapRenderer';
-import MapGeocoder from './MapGeocoder';
-import FeedbackGif from './FeedbackGif';
-import SignUpDialog from './SignUpDialog';
-
-export {
-  CardGrid,
-  QRGenerator,
-  PuzzleGenerate,
-  PuzzleInput,
-  MapRenderer,
-  MapGeocoder,
-  FeedbackGif,
-  SignUpDialog
-};
+export { default as CardGrid } from './CardGrid';
```

Comment:
Is this better form?

Reply:
It will not cause unnecessary merge conflict and it's easier to keep track of what function is being added